poker-pals/src/components/Poker.vue

```diff
@@ -1,14 +1,749 @@
+                            v-bind:timer="player.timer"
+                        />
+                    </div>
+                </div>
+                <div class="blindsBox">Blinds: 1,000/2,000</div> <!--shows the blinds of this particular table, later bind it to a variable-->
+            </div>
+            <div class="display">
+                <div class="cheatSheetToggle" v-on:click="toggleCheatSheet()"> <!--a toggle to show or hide the cheat sheet-->
+                    <img src="../images/cheat_sheet_icon.png"/>
+                </div>
+                <div class="cheatSheet" id="cheatSheet">
+                    <img src="../images/cheatSheet.png"/>
+                </div>
+                <div class="playerCards"> <!--container holding the players personal cards-->
+                    <div class="playerCard" id="playerCard1">
+                        <img v-bind:src="myCards[0].src"/>
+                    </div>
+                    <div class="playerCard" id="playerCard2">
+                        <img v-bind:src="myCards[1].src"/>
+                    </div>
+                </div>
+                <div class="playerInputs"> <!--container holding all the players button options-->
+                    <button class="InputButton1 centerText" id="buttonFold" v-on:click="makeDecision('FOLD')">FOLD</button>
+                    <button class="InputButton1 centerText" id="buttonCheck" v-on:click="makeDecision('CHECK')">CHECK</button>
+                    <button class="InputButton1 centerText" id="buttonCall" v-on:click="makeDecision('CALL')">CALL</button>
+                    <div class="raiseToggle"> <!--the container holding the raise input, it contains two buttons, a slider, and a value display-->
+                        <div class="raiseButtonOuter" id="buttonMinus" v-on:click="decrementRaise()">
+                            <div class="raiseButton centerText">-</div>
+                        </div>
+                        <div class="raiseValue" id="raiseValue">2000</div>
+                        <input class="raiseScroll" id="slider" type="range" min="2000" max="100000" value="2000">
+                        <div class="raiseButtonOuter" id="buttonPlus" v-on:click="incrementRaise()">
+                            <div class="raiseButton centerText">+</div>
+                        </div>
+                    </div>
+                    <button class="InputButton1 centerText" id="buttonRaise" v-on:click="makeDecision('RAISE')">RAISE</button>
+                    <button class="InputButton2 centerText" id="buttonCheckFold" v-on:click="toggleCheckFoldButton()">CHECK/FOLD</button>
+                    <button class="InputButton1 centerText" id="buttonAllIn" v-on:click="makeDecision('ALL IN')">ALL IN</button>
+                </div>
+            </div>
+        </div>
+        <Chat v-bind:full="chatFull"/> <!--the chat container holding all items related to messenging other players-->
```

Comment:
typo

Reply:
ah yes, my poor spelling skills are here to haunt me